components/connector-service/README.md

```diff
@@ -20,29 +20,29 @@ To start the Connector Service, run this command:
-- **connectorServiceHost** - Host under which this service is accessible. It is used for generating the URL. The default host is `cert-service.wormhole.cluster.kyma.cx`.
-- **gatewayBaseURL** - Base URL of the Gateway Service.
-- **certificateProtectedHost** - Host secured with the client certificate, used for the certificate renewal. The default host is `gateway.wormhole.cluster.kyma.cx`.
-- **appsInfoURL** - URL at which the management information for applications is available. If not provided, it bases on `connectorServiceHost`.
-- **runtimesInfoURL** - URL at which the management information for runtimes is available. If not provided, it bases on `connectorServiceHost`.
-- **appCertificateValidityTime** - Time until which the certificates that the service issues for Applications are valid. The default value is 90 days.
-- **runtimeCertificateValidityTime** - Time until which the certificates that the service issues for Runtimes are valid. The default value is 90 days.
-- **central** - Determines whether the Connector Service works in the central mode.
-- **revocationConfigMapName** - Name of the ConfigMap containing the revoked certificates list.
-- **lookupEnabled** - Determines if the Connector should make a call to get the gateway endpoint. The default value is`False`.
-- **lookupConfigMapPath** - Path in the Pod where Config Map for cluster lookup is stored. The default value is `/etc/config/config.json`. Used only when **lookupEnabled** is set to `True`
-
-Connector Service also uses following environmental variables for CSR - related information config:
+- **appName** is the name of the application used by Kubernetes deployments and services. The default value is `connector-service`.
+- **externalAPIPort** is the port that exposes the Connector Service API to an external solution. The default port is `8081`.
+- **internalAPIPort** is the port that exposes the Connector Service within Kubernetes cluster. The default port is `8080`.
+- **namespace** is where Connector Service is deployed. The default Namespace is `kyma-integration`.
+- **tokenLength** is the length of registration tokens. The default value is `64`.
+- **appTokenExpirationMinutes** is the time after which tokens for Applications expire and are no longer valid. The default value is `5` minutes.
+- **runtimeTokenExpirationMinutes** is the time after which tokens for Runtimes expire and are no longer valid. The default value is `10` minutes.
+- **caSecretName** is the Namespace and the name of the Secret which stores the certificate and the key used to sign client certificates. Requires the `Namespace/secret name` format. The default value is `kyma-integration/nginx-auth-ca`.
+- **rootCACertificateSecretName** is the Namespace and the name of the Secret which stores the root CA (Certificate Authority) Certificate in case the certificates are signed by the intermediate CA. Requires the `Namespace/secret name` format. 
```

Comment:
```suggestion
- **rootCACertificateSecretName** is the Namespace and the name of the Secret which stores the root CA (Certificate Authority) Certificate in case the certificates are signed by the intermediate CA. Requires the `Namespace/secret name` format. Empty by default. 
```